### PR TITLE
Update marshmallow.md

### PR DESCRIPTION
**Pull request by js6320**

### CODE REVIEW

**Comment by js6320** [comment]
Please review this, @casey-collab

File: korean/marshmallow.md [hunk 1]
```diff
@@ -12,4 +12,6 @@
 
 [어쩔줄 모르고 그냥 운다](cry/cry.md)?
 
-[마시멜로를 모은다](gather-the-marshmellows/gather-the-marshmellows.md)?
\ No newline at end of file
+[마시멜로를 모은다](gather-the-marshmellows/gather-the-marshmellows.md)?
+
+[시간을 확인한다](check-current-time/check-current-time.md)?
```
Comment (casey-collab) [inline korean/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**